src/app.js

```diff
@@ -63,6 +63,11 @@ const { app, router } = setup({
     public: "/public",
   },
   publicDirs: ["../dist/public"],
+  translation: {
+    allowedLangs: ["en", "cy"],
+    fallbackLang: ["en"],
+    cookie: { name: "lng" },
```

Comment:
Should the browser tests change as well, it would be appropriate to do so in this PR

Reply:
^ actually scratch that I've seen your comment on the other PR